tests/suite/editIdentityDepositorySuite.ts

```diff
@@ -8,13 +8,15 @@ import {
 import { getConnection, TXN_OPTS } from '../connection';
 import { editIdentityDepositoryTest } from '../cases/editIdentityDepositoryTest';
 
-export const editIdentityDepositorySuite = async function (
-  controllerAuthority: Signer,
-  user: Signer,
-  payer: Signer,
-  controller: Controller,
-  depository: IdentityDepository
-) {
+export const editIdentityDepositorySuite = async function ({
```

Comment:
imo we could dedicate a PR specifically to move to object-based parameters, this might be a bit too much to review at the same time as a on-chain change